torch/_ops.py

```diff
@@ -782,6 +782,20 @@ def tags(self):
     # TODO: add more methods to expose information about input and output arguments
 
 
+# TorchBindOpOverload are those custom ops which have at least one overload's
+# schema consists of torch.ScriptObject (i.e. custom class) input.
+# TorchBindOpOverload will skip C++ dispatcher and purely dispatched in python
+# when its inputs contain FakeScriptObject.
+class TorchBindOpOverload(OpOverload):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._is_torchbind_op = True
```

Comment:
Why do we need a `self._is_torchbind_op`? It seems like one can just do isinstance(self, torch._ops.TorchBindOpOverload)